clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientCloudBuilderStep1.java

```diff
@@ -44,7 +44,16 @@
        */
       ZeebeClientCloudBuilderStep4 withClientSecret(String clientSecret);
 
-      interface ZeebeClientCloudBuilderStep4 extends ZeebeClientBuilder {}
+      interface ZeebeClientCloudBuilderStep4 extends ZeebeClientBuilder {
+
+        /**
+         * Sets the region of the Camunda Cloud cluster. Default is 'bru-2'.
```

Comment:
We need to document what regions a user should put here, in the docs, or in console. Additionally, if we document it in the official docs page, I would also point it out here.

Reply:
I created an docs issue for this https://github.com/camunda-cloud/camunda-cloud-documentation/issues/302. I guess currently we don't expose the region in way in Console.